diktat-rules/src/main/kotlin/com/saveourtool/diktat/ruleset/rules/chapter3/LineLength.kt

```diff
@@ -458,14 +504,22 @@ class LineLength(configRules: List<RulesConfig>) : DiktatRule(
          * Abstract fix - fix anything nodes
          */
         abstract fun fix()
+
+        /**
+         * Abstract unFix - unfix fix-changes in anything nodes
+         */
+        abstract fun unFix()
```

Comment:
you can add a default implementation here (empty) and override only when it's required

Reply:
In this case, I can't add default implementation, because each `unFix()` method is unique and depends on operations in `fix()` method of each class.

Reply:
```kotlin
@Suppress("EmptyFunctionBlock")
override fun unFix() {}
```
this one is default implementation

Reply:
Done